src/envo/envo-edit.owl

```diff
@@ -47257,10 +47256,6 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_61452> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_61642> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_09200010> "soil pH"@en)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_09200022> "sunshine hours")
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_09200022> "envoCryo")
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_09200022> "envoPolar"^^xsd:string)
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_09200022> "hours of sunshine")
```

Comment:
This is a MAJOR change. The ID ENVO:09200022 was used for something entirely different (mass density of snow):

```
grep -r ENVO:09200022 modules/*.csv
modules/entity_attribute.csv:ENVO:09200022,,PATO:0001019,mass density,ENVO:01000406,snow
```

I do not know which ID has precedence here, the `mass density of snow` or the `sunshine hours`.

@pbuttigieg advice.

Reply:
neither were released but even so I would obsolete the ID and make two new ones just to be safe

Reply:
Thanks!

Reply:
Fixed in #1403